en_US/security/authz/redis.md

```diff
@@ -141,5 +141,5 @@ Otherwise the check returns immediately without proceeding to the next auth/ACL
 When the rule is non-empty and does not match the corresponding pub/sub permission,
 an authentication failure will be returned (the corresponding pub/sub behavior will be denied) and the authentication chain will be terminated.
 
-When more than one auth/ACL plugins are in use, it is recommended to position Redis ACL after other auth/ACL plugins in the enabled plugins list.
+When more than one auth/ACL plugins is in use, it is recommended to position Redis ACL after other auth/ACL plugins in the enabled plugins list.
```

Comment:
Why is this recommended?

Reply:
this was added because redis is a so called 'whitelist-only' auth backend, a 'nomatch' in this case is a 'deny'.
this statement should be generalised.